languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -97064,50 +97064,91 @@ The accident victim died from her injuries.
+                <token postag="RB" />
+                <token postag="SENT_END|\." postag_regexp="yes"/>
+            </antipattern>
+            <antipattern>
+                <token regexp="yes">&quote;</token>
+                <token chunk="B-ADVP" />
+                <token regexp="yes">&quote;</token>
+            </antipattern>
+            <antipattern>
+                <token skip="1" regexp="yes">\p{P}|\.\.\.</token>
+                <token chunk_re="[BI]-ADVP" />
+            </antipattern>
+            <antipattern>
+                <token regexp="yes">a|as|behave|big|is|least|mean|most|none|said|smiled|so|than|think|to|up|why|□|></token>
+                <token postag="RB" />
+            </antipattern>
             <antipattern>
                 <token postag="SENT_START" />
-                <token>maybe</token>
-                <token postag="SENT_END" />
+                <token postag="RB" />
+                <token min="2">.</token>
             </antipattern>
             <antipattern>
-                <token>,</token>
-                <token />
                 <token>maybe</token>
+                <token>.</token>
+                <token postag="PCT|CD" postag_regexp="yes"/>
             </antipattern>
             <antipattern>
-                <token regexp="yes">a|but|by|is|it|my|then|"</token>
-                <token>maybe</token>
+                <token skip="-1" regexp="yes" inflected="yes">answer|tell|run|say</token>
+                <token>honestly</token>
             </antipattern>
             <antipattern>
-                <token>maybe</token>
-                <token postag="JJ|VBD|RB|NN" postag_regexp="yes" />
+                <token min="0" regexp="yes">no|on</token>
+                <token regexp="yes">complete|one|purpose</token>
+                <token regexp="yes">certainly|apparently|obviously</token>
+            </antipattern>
+            <antipattern>
+                <token postag="VBG" />
+                <token>probably</token>
             </antipattern>
             <antipattern>
-                <token>“</token>
+                <token>always</token>
+                <token>be</token>
+                <token>my</token>
                 <token>maybe</token>
-                <token>”</token>
             </antipattern>
             <pattern>
-                <token postag=",|:|``|PCT" postag_regexp="yes" negate_pos="yes">
-                    <exception regexp="yes">also|least|mean|say|said|think|this|>|\(|-|\[|□|\.</exception>
-                </token>
-                <token>maybe</token>
-                <token postag="SENT_END">
-                    <exception regexp="yes">https.*</exception>
-                </token>
+                <token><exception regexp="yes">\p{P}</exception></token>
+                <token regexp="yes">actually|allegedly|apparently|certainly|fortunately|honestly|hopefully|however|maybe|obviously|perchance|perhaps|probably|surely|unfortunately</token>
```

Comment:
One last thing: could you place a `<marker>` only around the adverb, please? The suggestion will then be, e.g. `, maybe`. Making the rule only match the key word prevents overlapping matches.